clang/runtime/dpct-rt/include/math.hpp

```diff
@@ -43,6 +44,43 @@ inline bool isnan(const sycl::ext::oneapi::bfloat16 a) {
                            "sycl::ext::oneapi::experimental.");
 #endif
 }
+
+template <typename T>
+inline int64_t zero_or_signed_extent(T val, unsigned bit) {
```

Comment:
Please add comment for publich math API.  and can put zero_or_signed_extent() in detail namespace?

Reply:
done